app/authProvider.js

```diff
@@ -84,4 +84,17 @@ export class AuthProvider {
       post_logout_redirect_uri: `${appBaseUri}/signout-callback-oidc`,
     });
   }
+
+  authorise() {
+    return (req, res, next) => {
+      if (!req.user) {
+        req.headers.referer = `${appBaseUri}${req.originalUrl}`;
+        this.login()(req, res, next);
+      } else if (req.user && req.user.organisation) {
+        next();
+      } else {
+        throw new Error('Not authorised matey');
```

Comment:
is this the error message we are going with? 😄

Reply:
Yes, since we don't have a story to show a proper not authorised page. I rather it was something silly so it's obvious it needs changing.